lib/v2/hnu/careers.js

```diff
@@ -0,0 +1,29 @@
+const got = require('@/utils/got');
+
+module.exports = async (ctx) => {
+    const rootUrl = 'http://scc.hnu.edu.cn/';
+    const currentUrl = `${rootUrl}module/getcareers?start_page=1&type=inner&day=&count=20&start=1`;
+    const response = await got({
+        method: 'get',
+        url: currentUrl,
+    });
+    const company_list = response.data.data;
+
+    ctx.state.data = {
+        title: `湖南大学 校内招聘`,
+        link: currentUrl,
```

Comment:
`link` should link to a human-readable website instead of API response.

Reply:
Please take a look at the `link` definition in the [documentation](https://docs.rsshub.app/en/joinus/advanced-feed.html).

> The URL of the website associated with the feed, which should link to a **human-readable** website

In the [Outputting the RSS](https://docs.rsshub.app/en/joinus/new-rss/start-code.html#via-api-outputting-the-rss) example, even though we obtain the data through an API (the issue list), `ctx.state.data.link` still provides a link to a human-readable website that serves the API responses.

Reply:
I know that. but i hope i can show the important information at the feed. so I add more info in the item. if I only show the info which is in the docs, i think this feed isn't good RSS what i want to see.

If I must comply with the document rules, then I can modify it as needed.

Reply:
Please look at the [example](https://docs.rsshub.app/en/joinus/new-rss/start-code.html#via-api-outputting-the-rss) carefully.
You don't present the users with the [API response](https://api.github.com/repos/DIYgod/RSSHub/issues) when he/she uses the open related website of an RSS feed from their RSS reader which is basically a big pile of text. You present a human-readable [site](https://github.com/DIYgod/RSSHub/issues).